erica/erica_legacy/api/v1/endpoints/grundsteuer.py

```diff
@@ -24,7 +24,10 @@ def send_grundsteuer(grundsteuer: GrundsteuerPayload, include_elster_responses:
     """
     try:
         request = GrundsteuerRequestController(grundsteuer, include_elster_responses)
-        return request.process()
+        result = request.process()
+        if "transferticket" in result:
+            result["transfer_ticket"] = result.pop("transferticket")
```

Comment:
👍

Reply:
```suggestion
        if "transferticket" in result:
            result["transfer_ticket"] = result.pop("transferticket", None)
```
Just to not run into some error or is that an error that we want to see?

Reply:
Isn't the `if` in the line right before making sure that it is present? O.o

Reply:
Yes, I overlooked that